internal/testutil/test_hcls.go

```diff
@@ -329,6 +342,40 @@ func RHclAwsHostedConnection() RHclCloudRouterConnectionAwsResult {
 	}
 }
 
+// packetfabric_cloud_router_connection_port
+func RHclCloudRouterConnectionPort() RHclCloudRouterConnectionPortResult {
+
+	portDetails := CreateBasePortDetails()
```

Comment:
OBS: The reason for this `LINT` failure is due to a dependency on `packetfabric_port` PR. We'll need to move the function `CreateBasePortDetails` to a separated commit since it's being used by two different PRs.